### PR TITLE
[CI] update public s3 cache credentials

### PR DESCRIPTION
**Pull request by kiranparajuli589**
fixes https://github.com/owncloud/client/issues/9469

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/10874/5/1
:boom: The GUI tests failed.
GUI Logs: (https://cache.owncloud.com/public/owncloud/client/10874/guiReportUpload/index.html)            
Server Logs: ((https://cache.owncloud.com/public/owncloud/client/10874/guiReportUpload/serverlog.log)

**Comment by kiranparajuli589** [comment]
without AWS credentials, the upload fails with

```console
time="2022-02-24T07:20:17Z" level=warning msg="AWS Key and/or Secret not provided (falling back to ec2 instance profile)"
time="2022-02-24T07:20:17Z" level=info msg="Attempting to upload" bucket=****** endpoint="******" region=us-east-1
time="2022-02-24T07:20:17Z" level=info msg="Uploading file" bucket=****** name=/drone/src/test/guiReportUpload/serverlog.log target=/owncloud/client/10874/guiReportUpload/serverlog.log
time="2022-02-24T07:20:17Z" level=error msg="Could not upload file" bucket=****** error="NoCredentialProviders: no valid providers in chain. Deprecated.\n\tFor verbose messaging see aws.Config.CredentialsChainVerboseErrors" name=/drone/src/test/guiReportUpload/serverlog.log target=/owncloud/client/10874/guiReportUpload/serverlog.log
time="2022-02-24T07:20:17Z" level=fatal msg="NoCredentialProviders: no valid providers in chain. Deprecated.\n\tFor verbose messaging see aws.Config.CredentialsChainVerboseErrors"
```
https://drone.owncloud.com/owncloud/client/10874/5/14

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/10876/5/1
:boom: The GUI tests failed.
GUI Logs: (https://cache.owncloud.com/public/owncloud/client/10876/guiReportUpload/index.html)            
Server Logs: ((https://cache.owncloud.com/public/owncloud/client/10876/guiReportUpload/serverlog.log)

**Comment by kiranparajuli589** [comment]
> Results for **GUI-tests** https://drone.owncloud.com/owncloud/client/10876/5/1 boom The GUI tests failed. GUI Logs: (https://cache.owncloud.com/public/owncloud/client/10876/guiReportUpload/index.html) Server Logs: ((https://cache.owncloud.com/public/owncloud/client/10876/guiReportUpload/serverlog.log)

report working properly.

File: .drone.star [hunk 1]
```diff
@@ -584,10 +584,10 @@ def uploadGuiTestLogs():
         },
         "environment": {
             "AWS_ACCESS_KEY_ID": {
-                "from_secret": "cache_s3_access_key",
+                "from_secret": "cache_public_s3_access_key",
             },
             "AWS_SECRET_ACCESS_KEY": {
-                "from_secret": "cache_s3_secret_key",
+                "from_secret": "cache_public_s3_secret_key",
```
Comment (individual-it) [inline .drone.star:590]: do we anctually need those at all?

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/10877/5/1
:boom: The GUI tests failed.
GUI Logs: (https://cache.owncloud.com/public/owncloud/client/10877/guiReportUpload/index.html)            
Server Logs: ((https://cache.owncloud.com/public/owncloud/client/10877/guiReportUpload/serverlog.log)

File: .drone.star (hunk 1 above)
Comment (kiranparajuli589) [reply .drone.star:590]: yes, otherwise, upload will fail like mentioned here https://github.com/owncloud/client/pull/9470#issuecomment-1049591535

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/10879/5/1
:boom: The GUI tests failed.
GUI Logs: (https://cache.owncloud.com/public/owncloud/client/10879/guiReportUpload/index.html)            
Server Logs: ((https://cache.owncloud.com/public/owncloud/client/10879/guiReportUpload/serverlog.log)

**Review by individual-it — APPROVED** [review]

**Review by Talank — CHANGES_REQUESTED** [review]
LGTM but please make the base branch of PR `2.10` instead of `master`

**Comment by kiranparajuli589** [comment]
> LGTM but please make the base branch of PR `2.10` instead of `master`

I can make another pr for the `2.10` branch but @talank why use the outdated cache credentials even in `master`?

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/10880/5/1
:boom: The GUI tests failed.
GUI Logs: (https://cache.owncloud.com/public/owncloud/client/10880/guiReportUpload/index.html)            
Server Logs: ((https://cache.owncloud.com/public/owncloud/client/10880/guiReportUpload/serverlog.log)

**Comment by Talank** [comment]
changes in `2.10` will eventually be transferred to master. But we tend to maintain the single flow of changes in client. i.e `PR->2.10->master`

**Review by Talank — APPROVED** [review]

**Pull request merged**